debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MysqlDefaultValueIT.java

```diff
@@ -517,7 +517,13 @@ public void dateAndTimeTest() throws InterruptedException {
         //current timestamp will be replaced with epoch timestamp
         ZonedDateTime t5 = ZonedDateTime.ofInstant(Instant.EPOCH, ZoneOffset.UTC);
         String isoString5 = ZonedTimestamp.toIsoString(t5, ZoneOffset.UTC, MySqlValueConverters::adjustTemporal);
-        assertThat(schemaJ.defaultValue()).isEqualTo(isoString5);
+        try {
```

Comment:
I'm wondering whether we can handle these subtle differences a bit more elegantly. How about injecting the DB version as an environment variable and then deciding based on that which assertion to run?